annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FieldSetterTemplateParameters.java

```diff
@@ -0,0 +1,30 @@
+package com.exactpro.epfast.annotation.processing;
+
+import javax.lang.model.element.Element;
+import java.util.List;
+
+class FieldSetterTemplateParameters {
+    private Element fastType;
+
+    private TypeName typeName;
+
+    private List<FastFieldElement> fastFields;
+
+    public FieldSetterTemplateParameters(TypeName typeName, List<FastFieldElement> fastFields, Element fastType) {
```

Comment:
It's enough to pass just `FastTypeElement`. It has all information you need. Moreover it can be passed to mustache directly as it has all that you need.